interface/forms/requisition/new.php

```diff
@@ -136,64 +129,72 @@
       float: left;
 
   }
-  .barcode{
-      position: relative;
-      right: -380px;
-  }
+
 </style>
+    <title><?php echo xlt('Lab Requisition ') . text($lab); ?></title>
```

Comment:
sorry, just noted this. Need to remove the trailing space in the translation constant:
```
<title><?php echo xlt('Lab Requisition') . ' ' . text($lab); ?></title>
```